cmd/commands/root.go

```diff
@@ -37,12 +37,14 @@ func NewRootCommand(c *Context, name string) *cobra.Command {
 				conf.Read(configFile)
 			}
 
-			if baseDir == "" {
-				baseDir = conf.BaseDir
-			}
-			baseDir = os.ExpandEnv(baseDir)
-			if err := os.MkdirAll(baseDir, 0770); err != nil {
-				return err
+			if cmd.Use != "version" {
```

Comment:
Just a thought experiment for later:
The fact that all commands except **version** depend on `baseDir` is depicted here. This can change in the future, and is coded both in the command implementation as well as this `if` statement.

Reply:
As far as I know, the base directory is needed only for the watermark. Eugene added it, and I didn't look into it yet. I agree that it can be done differently rather than hard coding it into an if condition. But I didn't want to complicate it, and it can be taken care of when significant features are added to CLI.